From a3ea05ce1fa16b095efacc4cf4f47326f6c77cd9 Mon Sep 17 00:00:00 2001
From: "Le.Ma" <Le.Ma@amd.com>
Date: Mon, 14 Aug 2017 10:27:31 +0800
Subject: [PATCH 2506/2831] drm/ttm: Include <linux/version.h> in
 ttm_bo_driver.h

It's because the KERNEL_VERSION check is first introduced in the header file.

Change-Id: I2230db6a7efabf10b0206c2feedde99d97287773
Signed-off-by: Le.Ma <Le.Ma@amd.com>
Reviewed-by: Roger He <Hongbo.He@amd.com>
---
 include/drm/ttm/ttm_bo_driver.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/drm/ttm/ttm_bo_driver.h b/include/drm/ttm/ttm_bo_driver.h
index cb5072a..9902976 100644
--- a/include/drm/ttm/ttm_bo_driver.h
+++ b/include/drm/ttm/ttm_bo_driver.h
@@ -41,6 +41,7 @@
 #include <linux/fs.h>
 #include <linux/spinlock.h>
 #include <linux/reservation.h>
+#include <linux/version.h>
 
 #define TTM_MAX_BO_PRIORITY	4U
 
-- 
2.7.4

